### DISCUSSION TITLE
MVT catalog item not showing on map

### DISCUSSION THREAD

**Discussion by MaiaWilliams** [TerriaJS/terriajs#5809] [Q&A] [answered]
Hi,

I am having trouble displaying vector tile data in TerriaMap. I am not getting any error messages and the item is listed in the Workbench - but no data shows on the map. All other geojson and csv items are working fine. The tiled data is in epsg:4326 and was created from a shapefile using ogr2ogr. This vector tile data opens fine in QGIS.

Is there something obvious that I'm missing?
Thank you!

My vector tiles are here:
[TestTiles_4326.zip](https://github.com/TerriaJS/terriajs/files/7154123/TestTiles_4326.zip)

Here is my initilization file:
`{
  "homeCamera": {
      "west": 119.81,
      "south": -23.43,
      "east": 120.21,
      "north": -23.23
  },
  
  "catalog": [
		{
      "type": "mvt",
      "name": "Test MVT",
	  "id": "2",
	  "url": "test/Maia/TestTiles_4326/{z}/{x}/{y}.pbf",
	  "layer":"Test MVT",
          "lineColor": "yellow",
          "fillColor": "yellow",
	  "minimumZoom":0,
	  "maximumNativeZoom":9,
	  "maximumZoom": 20,
          "idProperty":"Site",
	  "nameProperty":"Name",
	  "keepOnTop": true,
	  "rectangle": {
					"west": 117,
					"south": -24,
					"east": 121,
					"north": -21						
					}
	},	
  ],
  "workbench": ["2"],

	/* "showSplitter": true,*/
	/* "splitPosition": 0.5,*/
   "viewerMode": "2D"
}
`
This is my item in the Workbench:
![image](https://user-images.githubusercontent.com/28177051/133058781-76625861-3806-4d56-9155-7f9b1a9287cd.png)

**Comment by tephenavies** [comment, marked as answer]
Hi @MaiaWilliams

I can think of a couple of problems you might be having:
1. Do your tiles have any polygons? The current MVT renderer is very basic and only renders polygons, so any points and lines are ignored. Way better MVT support is coming very soon!
2. Is "Test MVT" the name of the layer in the tiles? The "layer" parameter needs to exactly match the layer name in the tiles.

**Comment by MaiaWilliams** [comment]
Thank you! Your second suggestion has solved my problem and everything looks great now.

Thanks for the response.
